.gitignore

```diff
@@ -0,0 +1,3 @@
+*.log
```

Comment:
Mind keeping this out of the repo? I prefer to keep application-related logs in a global .gitignore

Reply:
Sure no problem.  Give me a second.

Reply:
All set 👍